packages/eslint-config-udemy-react-addons/README.md

```diff
@@ -1,4 +1,26 @@
-# ESLint Config Udemy React
+Udemy ESLint Config for React
```

Comment:
It says this is a "Config", but I think of them as "Add-ons" which supplement a larger config. Same comment for the other -addons directory.

Reply:
Well, this is still a config, that happens to be named as add ons. I'd like to stick to eslint's jargon here.